tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -1481,6 +1508,21 @@ class MklFusedConvOp
       OP_REQUIRES(context, num_args == 1,
                   errors::InvalidArgument(
                       "Fused Conv2D must have one extra argument: bias."));
+    } else if (fused_ops == std::vector<string>{"BiasAdd", "Add"}) {
+      this->set_fuse_biasadd(true);
+      this->set_fuse_add(true);
+      OP_REQUIRES(
+          context, num_args == 2,
+          errors::InvalidArgument(
+              "Fused Conv2D must have two extra arguments: bias and add."));
+    } else if (fused_ops == std::vector<string>{"BiasAdd", "Add", "Relu"}) {
+      this->set_fuse_biasadd(true);
+      this->set_fuse_add(true);
+      this->set_fuse_relu(true);
+      OP_REQUIRES(
+          context, num_args == 2,
```

Comment:
Shouldn't `num_args` be 3? Please also fix the error string below.

Reply:
@penpornk the num_args count the fused op that have additional input, relu did not have additional input, so the num_args is 2 for biasadd and add.